components/molecules/ContributionsTreemap/contributions-treemap.tsx

```diff
@@ -2,8 +2,8 @@ import { useSpring, animated } from "@react-spring/web";
 import dynamic from "next/dynamic";
 import { Datum } from "@nivo/line";
 import Card from "components/atoms/Card/card";
-import { SpecialNode } from "components/molecules/TreemapPrototype/special-node";
-import { ContributorNode } from "components/molecules/TreemapPrototype/contributor-node";
+import { SpecialNode } from "components/molecules/ContributionsTreemap/special-node";
```

Comment:
These have beem move to alongside the contributions treemap component where they're used.